src/templates/domain/useCases/tests/update.test.ejs

```diff
@@ -3,29 +3,26 @@ const assert = require('assert')
 const { <%= props.name.pascalCase %> } = require('../../entities')
 
 
-describe('Update <%= props.name.pascalCase %>', () => {
-  function aUser({ hasAccess }) {
-    return { hasAccess }
-  }
+describe('Update <%= props.name.raw %>', () => {
```

Comment:
It will need to be implemented in the future becase the `if (!ctx.user.isValid()) return Err.invalidEntity({` validation is applied for entities that has fields validation, this PR don't contemplate it.